src/sentry/uptime/subscriptions/regions.py

```diff
@@ -1,10 +1,16 @@
+from collections.abc import Sequence
+
 from django.conf import settings
 
+from sentry import options
 from sentry.conf.types.uptime import UptimeRegionConfig
 
 
 def get_active_region_configs() -> list[UptimeRegionConfig]:
-    return [v for v in settings.UPTIME_REGIONS if v.enabled]
+    configured_regions: Sequence[UptimeRegionConfig] = settings.UPTIME_REGIONS
+    disabled_region_slugs: Sequence[str] = options.get("uptime.disabled-checker-regions")
```

Comment:
Need to define this option

Reply:
Was done here https://github.com/getsentry/sentry/pull/85097